packages/webdriverio/tests/commands/element/moveTo.test.js

```diff
@@ -60,7 +60,7 @@ describe('moveTo', () => {
 
         const elem = await browser.$('#elem')
         request.setMockResponse([{}, {}])
-        expect(elem.moveTo(5, 10)).rejects.toThrow('Failed to getElementRect')
+        await expect(elem.moveTo(5, 10)).rejects.toThrow('Failed to receive element rects via execute command')
```

Comment:
This is currently broken due to the command not being awaited. This PR just exposes the broken UT.

Reply:
looks like my fault

Reply:
@abjerstedt thanks for fixing this one!